File: Programming-languages-and-laboratory/Python/Exercises/30_compito_java_20190617/Squadra.py
class Squadra:
    def __init__(self, codice, sport, nome, vinte, perse):
        self._codice = codice
        self._sport = sport
        self._nome = nome
        self._vinte = vinte
        self._perse = perse
        self._giocatori = []

    def add_giocatore(self, gioc):
        self._giocatori.append(gioc)

    def get_giocatori(self):
        return self._giocatori

    def get_nome(self):
        return self._nome

    def __str__(self):
        return self._nome + "\t" + str(self._codice) + "\t" + str(self._vinte) + "\t" + str(self._perse)

File: Programming-languages-and-laboratory/Java/Exercises/09_compito_java_2016_06_15/Squadra.java
import java.util.*;

public class Squadra {
    protected String nome;
    protected int codice;
    protected String sport;
    protected int vinte;
    protected int perse;
    protected List<Giocatore> giocatori;

    public Squadra(String nome, int codice, String sport, int vinte, int perse) {
        this.nome = nome;
        this.codice = codice;
        this.sport = sport;
        this.vinte = vinte;
        this.perse = perse;
        giocatori = new LinkedList<Giocatore>();
    }

    public void addGiocatore(Giocatore g) {
        giocatori.add(g);
    }

    public int nGiocatori() {
        return giocatori.size();
    }

    public int getCodice() {
        return codice;
    }

    public List<Giocatore> getGiocatori() {
        return giocatori;
    }

    public String getNome() {
        return nome;
    }

    public String toString() {
        return nome + "\t" + codice + "\t" + vinte + "\t" + perse + "\t";
    }
}